dotcom-rendering/src/web/browser/index.ts

```diff
@@ -3,13 +3,15 @@ import './webpackPublicPath';
 // these modules are bundled in the initial (i.e. this) chunk, so that they run ASAP
 import { bootCmp } from './bootCmp';
 import { dynamicImport } from './dynamicImport';
+import { ga } from './ga';
 import { islands } from './islands';
 import { ophan } from './ophan';
 import { sentryLoader } from './sentryLoader';
 import { startup } from './startup';
 
 startup('bootCmp', null, bootCmp);
 startup('ophan', null, ophan);
+startup('ga', null, ga);
```

Comment:
Should `ga` should be lower priority and hence initialised later than islands?

Reply:
i'm not sure - we run ophan asap so i moved it there. happy to move if it's the wrong place.

these are all async calls anyway, so they effectively run concurrently:

<img width="427" alt="image" src="https://user-images.githubusercontent.com/867233/216045271-8a27bf7f-7010-447d-beb3-51a39ba4d460.png">

GA needs consent before it even requests the library, so it won't be doing much till then:

<img width="330" alt="image" src="https://user-images.githubusercontent.com/867233/216045735-25672ccc-8dd3-4eed-95b6-c20ecd04ba75.png">